content/applications/marketing/email_marketing/analyze_metrics.rst

```diff
@@ -0,0 +1,111 @@
+===============
+Analyze metrics
+===============
+
+.. |CTR| replace:: :abbr:`CTR (Click through rate)`
+
+To properly understand the success or failure of an email marketing campaign, it is
+necessary to monitor several key metrics. The insights gained from these metrics can then be used to
+optimize future campaigns. Odoo's **Email Marketing** application tracks several :ref:`key metrics
+<email-marketing/key-metrics>`, that can be interpreted to improve future campaigns.
+
+View metrics
+============
+
+After a mass mail has been sent, the results for that particular mailing are displayed in multiple
+locations.
+
+To access the metrics for an individual mailing, navigate to :menuselection:`Email Marketing app
+--> Mailings`. Locate the specific mailing in the list view, and use the column headings to view the
+results for that mailing. Click on one of the mailings in the list to open the record. At the top of
+the record, detailed metrics are displayed as smart buttons.
+
+.. image:: analyze_metrics/metric-smart-buttons.png
+   :align: center
+   :alt: The smart buttons on a mass mailing, displaying the results of the message.
+
+.. tip::
+   Click on the smart buttons to see the recipient records for each metric.
+
+.. _email-marketing/key-metrics:
+
+Opened rate
+-----------
+
+The *Opened* rate measures the percentage of emails opened by recipients against the total number of
+sent emails.
```

Comment:
I totally understand the use of italics here, as this is, essentially, a definition -- which is totally fine. You can also consider to go the GUI route, and phrase it like: 
"The :guilabel:`Opened` smart button displays the percentage of ..."

But, honestly, either route is totally fine. Just figured it was worth mentioning for your consideration during your revision. 👍